docs/ert/reference/configuration/queue.rst

```diff
@@ -332,23 +295,6 @@ The following is a list of all queue-specific configuration options:
 
     QUEUE_OPTION TORQUE SUBMIT_SLEEP 0.5
```

Comment:
How does this differ from `SUBMIT_SLEEP` https://fmu-docs.equinor.com/docs/ert/reference/configuration/keywords.html#submit-sleep?

Reply:
After clarification, this will stay.